go.mod

```diff
@@ -11,121 +11,126 @@ require (
 	github.com/nlepage/go-tarfs v1.2.1
 	github.com/onsi/ginkgo/v2 v2.13.2
 	github.com/onsi/gomega v1.30.0
-	github.com/operator-framework/api v0.17.4-0.20230223191600-0131a6301e42
-	github.com/operator-framework/helm-operator-plugins v0.0.11
-	github.com/operator-framework/operator-registry v1.28.0
-	github.com/spf13/cobra v1.7.0
+	github.com/operator-framework/api v0.21.0
+	github.com/operator-framework/helm-operator-plugins v0.1.1
+	github.com/operator-framework/operator-registry v1.34.0
+	github.com/spf13/cobra v1.8.0
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.8.4
 	golang.org/x/crypto v0.18.0
 	golang.org/x/sync v0.4.0
-	helm.sh/helm/v3 v3.11.1
-	k8s.io/api v0.26.10
-	k8s.io/apiextensions-apiserver v0.26.10
-	k8s.io/apimachinery v0.26.10
-	k8s.io/cli-runtime v0.26.10
-	k8s.io/client-go v0.26.10
-	k8s.io/component-base v0.26.10
-	k8s.io/kube-aggregator v0.26.10
-	k8s.io/utils v0.0.0-20230220204549-a5ecb0141aa5
+	helm.sh/helm/v3 v3.13.3
+	k8s.io/api v0.28.5
+	k8s.io/apiextensions-apiserver v0.28.5
+	k8s.io/apimachinery v0.28.5
+	k8s.io/cli-runtime v0.28.5
+	k8s.io/client-go v0.28.5
+	k8s.io/component-base v0.28.5
+	k8s.io/kube-aggregator v0.28.5
```

Comment:
We should probably bump the KinD version to v0.20.0 and use a 1.28 based node image to ensure we are testing against a 1.28 cluster in e2e

Reply:
sgtm

Reply:
🤔 Something about this breaks the e2e test where we pull from a private registry. Investigating.

Reply:
I'll try this again after I rebase, but I'm tempted to separate this out into a separate PR if I can't get it working immediately.

Reply:
https://github.com/operator-framework/rukpak/issues/803